datastream-server-api/src/main/java/com/linkedin/datastream/server/api/connector/DatastreamChangeListener.java

```diff
@@ -48,4 +48,12 @@ default void handleDatastream(List<DatastreamGroup> datastreamGroups) {
   default Map<String, Optional<DatastreamGroupPartitionsMetadata>> getDatastreamPartitions() {
     return new HashMap<>();
   }
+
+  /**
+   * Get if the partition management is supported. Return true if the connector can rely on Brooklin to management
```

Comment:
I think we need to improve and expand this javadoc a bit. This is a fairly critical feature and it deserves more explanation.

Reply:
+1

Reply:
Thanks for the suggestion. Will add some clarification.